### ISSUE TITLE
Installs and opens, but can't give any commands

### ISSUE THREAD

**Issue by sixtyfive** [mauvilsa/nw-page-editor#6]
Hi, this might be a severe case of pebkac, in which case I apologize. After installing and opening nw-page-editor (trying with one of my own scans as well as your examples) I can neither create not modify any of the polygons. None of the commands from the README (except for Ctrl+O) seem to do anything at all. Wondering if the NW.JS version (83.0.4103.61) might be the problem?

**Comment by mauvilsa** [comment]
Hi @sixtyfive. On which operating system are you trying? I have tested the desktop app on mac and on linux with the latest version of nw.js and it works without issues. Note that there are many edit modes, so you need to enable the mode for modifying the coordinates of the elements you wish. Open the menu by clicking on the three horizontal lines at the top right to see all of the edit modes that the app supports. For example, open the `examples/lorem.xml` example and then enable the mode TextRegion-Coords. If you click on a text region you should be able to modify it.

**Comment by sixtyfive** [comment]
Hi @mauvilsa. I first tried on Windows, but double-clicking the .exe only resulted in a short flash of perhaps a window showing up and then nothing. Next attempt was with ArchLinux (with the nw.js binary version from AUR), where I observed the behaviour described. Now trying with Solus and latest nw.js downloaded from their site. Now everything works as expected. I'll try again on Monday with the Arch box and the same nw.js version as here at home. Thank you!
